Sources/Plasma/FeatureLib/pfGameGUIMgr/pfGUIEditBoxMod.cpp

```diff
@@ -456,56 +399,26 @@ bool    pfGUIEditBoxMod::HandleKeyEvent( pfGameGUIMgr::EventType event, plKeyDef
     }
 }
 
-std::string pfGUIEditBoxMod::GetBuffer()
-{
-    char* temp = hsWStringToString(fBuffer);
-    std::string retVal = temp;
-    delete [] temp;
-    return retVal;
-}
-
 void    pfGUIEditBoxMod::ClearBuffer()
 {
-    if (fBuffer != nullptr)
-    {
-        memset( fBuffer, 0, (fBufferSize + 1) * sizeof(wchar_t) );
-        fCursorPos = 0;
-        fScrollPos = 0;
-        IUpdate();
-    }
-}
-
-void    pfGUIEditBoxMod::SetText( const char *str )
-{
-    wchar_t* temp = hsStringToWString(str);
-    SetText(temp);
-    delete [] temp;
+    fBuffer.allocate(fBuffer.size(), 0);
```

Comment:
This will force a reallocation...  Unfortunately, `ST::buffer` doesn't have a zero operation currently, so I think the memset is still the best option:
```suggestion
    memset(fBuffer.data(), 0, fBuffer.size());
```